README.md

```diff
@@ -64,8 +64,20 @@ You can customize the pre-defined lint sets, both to disable one or more of the
 lints included, or to add additional lints. For details see [customizing static
 analysis].
 
+## Migrating from the pedantic package
+
+In migrating from the pre-defined lint sets provided by the [pedantic package],
+you get additional static analysis coverage from 29 additional lints.  There
+are, however, 6 lints which the pedantic package includes which are not
+included in the pre-defined lint sets provided by the lints package:
+`always_declare_return_types`, `prefer_single_quotes`,
+`sort_child_properties_last`, `unawaited_futures`, `unsafe_html`, and
+`use_full_hex_values_for_flutter_colors`. To keep any of these lints enabled,
```

Comment:
Maybe we can point out that `sort_child_properties_last` and `use_full_hex_values_for_flutter_colors` are now in `flutter_lints`?

Reply:
Done